sechub-server/src/test/resources/application-test.properties

```diff
@@ -93,4 +93,5 @@ sechub.notification.email.administrators=sechub-admin-test@example.org
 
 # scheduler
 sechub.config.trigger.nextjob.initialdelay=100
-sechub.config.trigger.nextjob.delay=300
\ No newline at end of file
+sechub.config.trigger.nextjob.delay=300
+sechub.scheduler.strategy.id=only-one-scan-per-project-at-a-time
```

Comment:
Hmm... those will not be used in integration tests, so only available in junit-tests. If there is a need keep it, otherwise this can be removed.

Reply:
done